aldryn_reversion/utils.py

```diff
@@ -27,7 +27,7 @@ def build_obj_repr(obj):
     Returns a unicode string of object Model name and its text representation.
     """
     return "{0}: '{1}'".format(force_text(obj._meta.model.__name__),
-                                force_text(obj))
+                               force_text(obj))
```

Comment:
Actually I'm a little bit confused with those forces, https://github.com/django/django/blob/0cbe3c63af72233877e95ef24fe571cecc7cad9f/django/utils/encoding.py#L163
What is the best practice for forcing text repr across misc django versions? Also it seems that those force_unicode = force_text is true for django versions 1.6, 1.7 and 1.8...